client.go

```diff
@@ -278,7 +278,7 @@ func (c *Client) GetRoom(ctx context.Context, roomID string) (Room, error) {
 }
 
 // GetRooms retrieves a list of rooms based on the options provided.
-func (c *Client) GetRooms(ctx context.Context, options GetRoomsOptions) ([]Room, error) {
+func (c *Client) GetRooms(ctx context.Context, options GetRoomsOptions) ([]core.RoomWithoutMembers, error) {
```

Comment:
This is fine here, but SDK users can't access the struct since it is part of the `internal` package. You'll have to alias this in the main package so it can be imported.